.eslintrc

```diff
@@ -4,6 +4,7 @@
   },
   "env": {
     "node": true,
-    "mocha": true
+    "jest": true,
+    "jasmine": true
```

Comment:
I think we can remove this, I added is because this reason:
> Previously a timeout of 10 seconds was set globally for all tests, I moved it locally to just babel-cli tests: https://github.com/babel/babel/compare/7.0...montogeek:feat/migrate-test-to-jest?expand=1#diff-ab3a365851f93959d9be46529b9b1500R147

But I [made](https://github.com/babel/babel/pull/5359/commits/fb5626beef6955c82a738ce837bd4423a95dba66) that config global and forgot to change it, thanks for pointing it out!